packages/mui-material/src/Grid/Grid.js

```diff
@@ -25,14 +25,6 @@ import useTheme from '../styles/useTheme';
 import GridContext from './GridContext';
 import gridClasses, { getGridUtilityClass } from './gridClasses';
 
-function getOffset(val) {
-  const parse = parseFloat(val);
-  if (Number.isNaN(parse)) {
-    return val;
-  }
-  return `${parse}${String(val).replace(String(parse), '') || 'px'}`;
-}
```

Comment:
I believe we can drop this since we have this new constraint: https://mui.com/material-ui/migration/v5-style-changes/#%E2%9C%85-remove-px-suffix.

It seems impossible to have values without units: https://github.com/mui/material-ui/blob/1ee901b29abe22e3c369d793c24fd1bdcff35ab7/packages/mui-system/src/createTheme/createSpacing.ts#L56